public/app/features/explore/QueryLibrary/QueryLibraryAnalyticsEvents.ts

```diff
@@ -0,0 +1,50 @@
+import { reportInteraction } from '@grafana/runtime';
+
+const QUERY_LIBRARY_EVENT = 'query_library_clicked';
```

Comment:
maybe we should add explore here?

Reply:
that's a very good point! especially because we are planning to have query library in dashboards, so it will be nice to have differentiators of explore, dashboard, app. This might require some refactoring in the future where the event title is passed as a param to these functions, but I'm not worried about it right now